modules/cert-manager-override-scheduling.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-override-scheduling_{context}"]
+= Configuring Scheduling Overrides for {cert-manager} Components
+
+After installing the {cert-manager-operator}, you can configure the scheduling of {cert-manager} components by configuring the `overrideScheduling` field in the {cert-manager} custom resource. This allows you to define `nodeSelector` and `tolerations` for the `controllerConfig`, `webhookConfig`, and `cainjectorConfig` sections.
```

Comment:
same here, while `{cert-manager-operator}` does the other `{cert-manager}` stays as is in the doc page.

Reply:
```suggestion
After installing the {cert-manager-operator}, you can configure the pod scheduling from the {cert-manager-operator} API for the cert-manager components such as cert-manager controller, CA injector, and Webhook. Configuring the `overrideScheduling` block in the certmanager.operator custom resource allows you to define `nodeSelector` and `tolerations` for the `controllerConfig`, `webhookConfig`, and `cainjectorConfig` sections respectively.
```